app/renderer/js/utils/dnd-util.ts

```diff
@@ -37,12 +43,128 @@ export function toggle(): Toggle {
 		ConfigUtil.setConfigItem('dndPreviousSettings', oldSettings);
 	} else {
 		newSettings = ConfigUtil.getConfigItem('dndPreviousSettings');
+		ConfigUtil.setConfigItem('dndSwitchOff', null);
+		if (job !== null) {
+			job.cancel();
+		}
 	}
 
 	for (const settingName of dndSettingList) {
 		ConfigUtil.setConfigItem(settingName, newSettings[settingName]);
 	}
-
 	ConfigUtil.setConfigItem('dnd', dnd);
 	return {dnd, newSettings};
 }
+
+export const makeView = (): void => { // Makes the DND Menu
+	const actionContainer = document.querySelector('#actions-container');
+	const dndExist = document.querySelectorAll('.dnd-menu');		// Check if the dnd menu is already created
+	if (dndExist.length === 0) {
+		const dndMenu = document.createElement('ul');
+		dndMenu.className = 'dnd-menu';
+		const dndOptions = [['30 Minutes', 'thirty_min'],
```

Comment:
@manavmehta @abhigyank this can be improved I think.

Reply:
@akashnimare you mean creating the menu ?

Reply:
@manavmehta I think it'd be better to use a `map` here - 
```js
let dndOptions = new Map();
dndOptions.set('thirty_minutes', '30 Minutes');
dndOptions.set('one_hour', '1 Hour');
...
```
And in the below iteration it'd be good to use `forEach` - 
```js
dndOptions.forEach(function(value, key) {
  ...
})
```

Reply:
used an object as akash suggested
if you find map more useful I can do the necessary change